supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -866,6 +868,33 @@ class GcpBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
     outputs.toSet ++ additionalGlobOutput
   }
 
+  /*
+    This method generates a deterministic job ID for the GCP Batch job using the short workflow id,
+    sanitized call name with appended hash, scatter index and attempt.
+    Note: Job ID needs to follow regex: ^[a-z]([a-z0-9-]{0,61}[a-z0-9])?$
```

Comment:
It would be helpful, either here or in the changelog, to include a "looks like" example.

Reply:
That is a really good point. I have added it to both places since the audience is different in each place.